docker/access-point/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM resin/rpi-raspbian
+
+RUN apt-get --yes update \
+    && apt-get --yes upgrade
+
+RUN apt-get --yes install \
```

Comment:
we always want the `apt-get update` and the `apt-get install` to be part of the same command, at least in the final version of the image.  for testing this is fine

Reply:
But... what if each Dockerfile has update and upgrade as a single command? Wouldn't this make on-pi builds faster, because this step is time consuming and would be cached after the first container does this step?

Reply:
totally, its fine for testing, but typically in a real-world scenario (e.g., work) you don't want this because you pretty much won't ever pull updates after that first layer is cached.  If there's a security vulnerability, you want to get those updates

building the images on the pi is really only the initial part.  we should push the containers to a registry and pull them to the other devices which will be way way faster